ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+## Description
+
+Provide a description of your issue.
+For any suggestions, please address them on the [forum](https://forum.freegamedev.net/viewforum.php?f=16).
+
+## Steps to reproduce
+
+## Configuration
+
+STK release version:
+
+STK Source (ppa, distribution package, official bin,...):
+If you did not download STK from the official website, please specify where you got it:
+
+System: 
+
+Graphics card: 
+
+CPU: 
+
+Gamepads/keyboards models if related to the issue:
+
+## Additional informations
+
+Please provide stdout.log, it is located in
+
+* %appdata%\supertuxkart\0.8.2 (windows)
+* ~/Library/Application Support/supertuxkart/0.8.2 (mac)
+* ~/.config/supertuxkart/0.8.2 (linux, and other unix based systems)
```

Comment:
Still incorrect, unix uses "$XDG_CONFIG_HOME/". Maybe just refer to the FAQ entry...

Reply:
Incorrect but second choice on the FAQ and on most linux machines it will resolve to the same path but ok I will add it.